src/directives/copy-to-clipboard.js

```diff
@@ -0,0 +1,17 @@
+import copy from 'clipboard-copy';
+
+/**
+ * The following directive attaches an event listener
+ * to the element, and copies the value provided
+ * to the directive
+ * @return {Object}
+ */
+export default {
+  inserted: (el, binding) => el.addEventListener('click', () => copy(binding.value).then(() => {
```

Comment:
Why don't use `await` here?
```suggestion
  inserted: (el, binding) => el.addEventListener('click', async () => {
    await copy(binding.value);
```

Reply:
@davidyuk doesn't work.

Reply:
Why you have set `regenerator` filed to false in `.babelrc`?
4a3f2c8e81504975d8fda56a285edabdfe794994

Reply:
@davidyuk I don't remember the reason, though thanks for the update. I'll merge the branch now.